src/otx/algorithms/detection/configs/instance_segmentation/maskrcnn_swin_t/hpo_config.yaml

```diff
@@ -0,0 +1,16 @@
+metric: mAP
+search_algorithm: asha
+early_stop: None
+hp_space:
+  learning_parameters.learning_rate:
+    param_type: qloguniform
+    range:
+      - 0.0001
+      - 0.01
+      - 0.0001
+  learning_parameters.batch_size:
+    param_type: qloguniform
+    range:
+      - 2
+      - 6
```

Comment:
Is batch size 8 too big to run on general device? If not, how about to change this value to 8?

Reply:
Updated